resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/engines/ApacheHttpAsyncClient4Engine.java

```diff
@@ -878,4 +877,10 @@ private static IOException ioException(Exception ex) {
       return (ex instanceof IOException) ? (IOException) ex : new IOException(ex);
    }
 
+   private static void safeClose(final Closeable closeable) {
+      if (closeable != null) try {
```

Comment:
I know it is just matter of style but using `if` without brackets could be a bit cryptic, especially with try-catch